src/pages/Models/Details/ModelDetails.js

```diff
@@ -179,16 +194,16 @@ const ModelDetails = () => {
               filters={viewFilters}
               setFilterToggle={setViewFilterToggle}
               filterToggle={viewFilterToggle}
+              disabled={slidePanelActive}
```

Comment:
This is one way of doing it, but why not just not render it at all? 🤷

Reply:
Because then it doesn't slide, it just appears.